config/helm/appmesh-controller/test.yaml

```diff
@@ -124,6 +124,10 @@ tracing:
   port: 2000
   # tracing.samplingRate: X-Ray tracer sampling rate
   samplingRate: 0.05
+  # tracing.logLevel: X-Ray agent log level
+  logLevel: ""
+  # tracing.role: X-Ray agent IAM role to upload segments to a different account
+  role: ""
```

Comment:
Would we want to set a default value for these?

Reply:
Xray agent's logLevel is by default `prod` [[ref](https://github.com/aws/aws-xray-daemon/blob/master/README.md?plain=1#L60)] which is same as `info`. I originally though of setting `role: "prod"` as default where it will enforce Xray agent's container Args "--log-level prod".

Intentionally I didn't set this value for 2 reasons:
1. Avoid unnecessarily setting this args on the Xray agent container
2. Avoid making changes in xray_test.go to update all the unit tests that should consider this args by default.

&nbsp;

For role we don't have a default value.

Reply:
Gotcha. Makes sense.

Reply:
Small change, I set `prod` as default logLevel. In switch case I will ignore this `prod` value.